### ISSUE TITLE
Upload functionality?

### ISSUE THREAD

**Issue by MichaelCurrie** [lsst-sqre/zenodio#4]
http://zenodio.lsst.io/en/latest/index.html says:

> We’re still designing the upload API, but metadata harvesting is ready to go.

Does this mean that uploading is not currently possible to Zenodo using the `zenodo` Python package?

Thanks.

**Comment by jonathansick** [comment]
That's right, I haven't implemented this yet (project's the backburner). Sorry!

**Comment by MichaelCurrie** [comment]
I will try to work on this and submit a PR.  I found this note from you at [a hackathon in January 2016](http://www.astrobetter.com/wiki/AASHackDay), so perhaps you will approve!

> Help build Zenodio (http://zenodio.lsst.io). Our goal is to build a sweet Python package for consuming information from Zenodo, and easily uploading new artifacts to Zenodo. My goal is to work on the upload API. One use case is uploading artifacts for a conference where editors have a base set of metadata that the participants can extend for their own uploads. (Sick)

**Comment by jonathansick** [comment]
That was the plan at the time 😆. My intention there was to build Python classes that abstracted API objects and took care of the serialization to the Zenodo API.

Since then, I'm starting to take a different strategic track and think about using [codemeta JSON-LD](https://github.com/codemeta/codemeta) as a primary metadata format, and then cross-walk that metadata to Zenodo's schema for submission. I think this is what we'll do internally, and will probably happen in a different repo. It might use Zenodio to do the final submission step.

The original plan, to build a dedicated API upload interface in zenodio is still probably a valid idea. If you want to work on this, feel free. I look forward to it!

**Comment by MichaelCurrie** [comment]
OK I forked the repo and [did some work](https://github.com/MichaelCurrie/zenodio/commit/f35dfc0c7c4a9469c928b1b333fbc6e69d99988f) but I see that you had started on an upload_api branch.  Is that still valid to build from or do you think that approach is no good?

**Comment by jonathansick** [comment]
That's right. I don't think there's anything useful on `upload_api`.
